.github/workflows/install.yaml

```diff
@@ -19,15 +39,116 @@ jobs:
         uses: azure/setup-helm@v3
         with:
           version: v3.10.0
-          
-      - name: Set up chart-testing
-        uses: scrungus/chart-testing-action@v3.7.4
       
       - name: Copy cloud creds to file
         run: 'echo "$TYLER_CLOUDS" > clouds.yml'
         shell: bash
         env:
           TYLER_CLOUDS: ${{ secrets.TYLER_CLOUDS }}
       
-      - name: Run Helm Install
-        run: ct install --target-branch ${{ github.event.repository.default_branch }} --all --extra-values clouds.yml --helm-extra-args "--timeout 5m" --openstack-deploy
+      - name: Create k8s Kind Cluster
+        uses: helm/kind-action@v1.5.0
+      
+      - name: Install cert-manager
+        run: >
+          helm repo add jetstack https://charts.jetstack.io
+          && helm repo update 
+          && helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --version v1.10.1 --set installCRDs=true --wait
+      
+      - name: Make kustomization directory 
+        run: mkdir clusterapi
+      
+      - name: Overwrite file
```

Comment:
lets add a comment saying it comes from here: https://github.com/stackhpc/ansible-collection-azimuth-ops/blob/main/roles/clusterapi/defaults/main.yml